rollup.config.js

```diff
@@ -47,7 +47,7 @@ export default [
   {
     ...cfg,
     output: {
-      file: './build/extension/holoplay-webxr-document_start.js',
+      file: './build/extension/lookingglass-webxr-document_start.js',
```

Comment:
for standardizing, this may also want to be `looking-glass-webxr-document-start.js`

Reply:
What's your hesitation in making the change? Makes sense to me for it to have the extra hyphen!

Reply:
no hesitation, just something I caught after seeing all the commits in one place 👍